pkgs/by-name/be/better-commits/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, nodejs 
+, nix-update-script
+}:
+
+buildNpmPackage rec {
+  pname = "better-commits";
+  version = "1.16.1";
+
+  src = fetchFromGitHub {
+    owner = "Everduin94";
+    repo = pname;
+    rev = "v${version}"; 
+
+    hash = "sha256-70PEcktGGck7egDmaIteFEMCNpo6ZuWyIVPiOywr2tc=";
+  };
+
+  npmDepsHash = "sha256-g34UutgT5315BpsQSuGGLIU6Ga+hpEz74HNLKKOB+ec=";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "A CLI for creating better commits following the conventional commits specification";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes
> Not start with the definite or an indefinite article.

Dropping the A and just doing "CLI for creating better commits following the conventional commits specification" should be fine. It's a funky requirement to get used to